lib/uinta/plug.ex

```diff
@@ -156,6 +157,13 @@ if Code.ensure_loaded?(Plug) do
         status: Integer.to_string(conn.status),
         timing: formatted_diff(diff),
         duration_ms: diff / 1000,
+        client_ip: conn.remote_ip,
+        user_agent: get_first_value_for_header(conn, "user-agent"),
```

Comment:
Shouldn't stuff like this just come from the logger metadata for apps that want it?

Reply:
I could make them optional but that s more inline with the default of Apache log and would avoid creating a plug?

Reply:
:shrug: I don't know, kinda on the fence here

I'm not seeing any metadata in here yet; what if had a nested map with logger metadata and provided a config option to limit what metadata is included (similar to [Logger.Backends.Console](https://hexdocs.pm/logger/Logger.Backends.Console.html) configuration)?

Then any app using Uinta could pass in any information they want--it would just have to put the desired information in the logger metadata. So it's just a question of balancing flexibility and convenience.

Reply:
I've been thinking about this more, and it's very request-specific data that doesn't make sense to include in the logger metadata. I'm okay with this.